source/_patterns/02-molecules/article/figure.json

```diff
@@ -1,5 +1,5 @@
 {
-  "figure_classes": "u-alignright u-size--medium",
+  "figure_classes": "u-size--medium",
```

Comment:
what does `.u-size--medium` control?

Reply:
This is a remnant from my standard baseline css styles. This, in combination with the alignment classes, allow editors to choose the alignment and sizing of inline media objects within the article body. Since the inline media objects for the NYPR themes would likely always span the article body column, I removed the styles. I'll remove this class assignment as well. 

What those classes look like in a more dynamic article body layout:

![img](https://d.pr/i/h3rohB+)